openmc/material.py

```diff
@@ -141,9 +138,9 @@ def __repr__(self):
         string += '{0: <16}\n'.format('\tElements')
 
         for element in self._elements:
-            percent = self._nuclides[element][1]
-            percent_type = self._nuclides[element][2]
-            string += '{0: >16}'.format('\t{0}'.format(element))
+            percent = self._elements[element][1]
```

Comment:
The fact this bug wasn't discovered until now indicates just how far we have to go to encourage adoption of the API!